packages/text-area/src/TextArea/TextArea.spec.tsx

```diff
@@ -20,14 +17,33 @@ const defaultProps = {
 };
 
 function renderTextArea(props = {}) {
-  const utils = render(
-    <TextArea data-testid="text-area" label={labelProp} {...props} />,
-  );
-
-  const textArea = utils.getByTestId('text-area');
-  const label = utils.container.querySelector('label');
-  const description = utils.container.querySelector('p');
-  return { ...utils, textArea, label, description };
+  const renderUtils = render(<TextArea label={labelProp} {...props} />);
+
+  const {
+    elements: { getDescription, getErrorMessage, getInput, getLabel },
+    utils: { isDisabled, isError, inputValue },
+  } = getLGTextAreaUtils();
+
+  const textArea = getInput();
+  const label = getLabel();
```

Comment:
can we test how you handle when there are multiple on the same page - how are we making sure the label/description belong to this input?